Adding the documentation of the new method of biometric plugin (#3182)
tauri-apps/tauri-docs: src/content/docs/plugin/biometric.mdx

</TabItem>
</Tabs>

> ### Biometric protected cryptography
> 

Reviewer: ```suggestion
### Biometric protected cryptography

:::caution [Android-only feature]
:::

```
is this android only? I think it would be good to warn the user, see more styles here if you think another one fits better https://starlight.astro.build/guides/authoring-content/#note-aside, feel free to propose an alternative.

Author: Yes, it's working only with Android for now. I'll take a look. Thanks ;-)

> To encrypt/decrypt data using an assymetric cryptography method that is protected behid the user Biometric Authentication, utilize the `biometricCipher()` method.

Reviewer: ```suggestion
To encrypt/decrypt data using an asymmetric cryptography method that is protected behind the user Biometric Authentication, utilize the `biometricCipher()` method.
```

<Tabs syncKey="lang">

<TabItem label="JavaScript">

> ```javascript ins={18}

Reviewer: ```suggestion
```javascript
```
I assume those aren't intended?

Author: Not really, just copied from the authenticate()'s snippet above.

import { biometricCipher } from '@tauri-apps/plugin-biometric';

// Encrypts data
const encryptOptions = {
  // ... other options
  dataToEncrypt: getOriginalData(),
};

try {
  const encrypted = await biometricCipher(
    'Passwordless authentication',
    encryptOptions
  );
  console.log(
    'Hooray! Successfully encrypted data! We can now store it to decrypt later, when needed'
  );
} catch (err) {
  console.log('Oh no! Authentication failed because ' + err.message);
}

// Decrypts data back to the original
const decryptOptions = {
  // ... other options
  dataToDecrypt: encrypted.data,
};

try {
  const original = await biometricCipher(
    'Passwordless authentication',
    decryptOptions
  );
  console.log(
    'Hooray! Successfully decrypted data after the user authenticated with their biometric method.'
  );
  const valid = originalData() == dataToDecrypt.data;
} catch (err) {
  console.log('Oh no! Authentication failed because ' + err.message);
}
```

</TabItem>

<TabItem label="Rust">

> ```rust ins={21}

Reviewer: ```suggestion
```rust
```
same
